installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc

```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports both x86-64 and AArch64 CPU architectures.
+You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports x86-64, AArch64, ppc64le, s390x and arm64 CPU architectures.
+
```

Comment:
Are AArch64 and arm64 CPU architectures considered distinct?

Reply:
@kelbrown20 Do you know when to use arm64 and when AArch64? I asume it's the same but we should probably only have the architecture label for ARM once.

Reply:
Thanks @kelbrown20. Updated as mentioned above.